packages/css/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@decathlon/vitamin-css",
+  "version": "0.0.1",
+  "description": "CSS styles for Vitamin Web",
+  "author": "Decathlon <designsystem@decathlo>",
+  "homepage": "https://github.com/Decathlon/vitamin-web#readme",
+  "scripts": {
+    "build": "cross-env NODE_ENV=production postcss src/index.css -o lib/index.css",
+    "storybook": "npm run build && start-storybook -s ./lib -p 6006",
```

Comment:
I think your system does not support watch mode. I suggest you split your scripts like this:
{
...
"build": "cross-env NODE_ENV=production npm run build:tailwind",
"build:tailwind": "postcss src/index.css -o lib/index.css",
"watch:tailwind": "chokidar 'src/**/*.css' -c 'npm run build:tailwind'",
 "storybook": "npm-run-all build:tailwind --parallel watch:tailwind start:storybook",
"start:storybook": "start-storybook -p 6006 -s public",
...  
}

Reply:
@gabbloquet @jleveugle @thollander if you have an idea... (: 
thx!

Reply:
I think it's because the css file is considered a static file. I think it must be imported into a decorator (global decorator, see my previous comment) to activate the hot reload...

Reply:
waiting for #20... thanks for @jleveugle:
using Postcss 7 instead of Postcss 8 (more stability for the watch mode)
yarn instead of npm (I think we need to discuss this)